src/types/__snapshots__/resolveStatements.spec.ts.snap

```diff
@@ -580,6 +580,36 @@ Line 7, col 9:
 "
 `;
 
+exports[`resolveStatements should fail statements for var-underscore-name-access 1`] = `
+"<unknown>:6:16: Unable to resolve id _
+Line 6, col 16:
+  5 |     foreach (_, _ in m) {
+> 6 |         return _;
+                     ^
+  7 |     }
+"
+`;
+
+exports[`resolveStatements should fail statements for var-underscore-name-access2 1`] = `
+"<unknown>:7:14: Unable to resolve id _
```

Comment:
I would expect this error message to be

> Variable name cannot be '_'

or, if we converge on the proposed change below:

> Wildcard variable name '_' cannot be used